profile/README.md

```diff
@@ -0,0 +1,90 @@
+
+For these crates we hold ourselves to high standards relating to security, testing, release
+schedule, API design, and coding style. Our current focus is on stabilising these crates [0].
+
+Name | Crate | Repository
+---|---|---
+base58 | [base58ck](https://crates.io/crates/base58ck) | [rust-bitcoin](https://github.com/rust-bitcoin/rust-bitcoin)
+bitcoin | [bitcoin](https://crates.io/crates/bitcoin) | [rust-bitcoin](https://github.com/rust-bitcoin/rust-bitcoin)
+bitcoinconsensus | [bitcoinconsensus](https://crates.io/crates/bitcoinconsensus) | [rust-bitcoinconsensus](https://github.com/rust-bitcoin/rust-bitcoinconsensus)
+bech32 | [bech32](https://crates.io/crates/bech32) | [rust-bech32](https://github.com/rust-bitcoin/rust-bech32)
+hashes | [bitcoin_hashes](https://crates.io/crates/bitcoin_hashes) | [rust-bitcoin](https://github.com/rust-bitcoin/rust-bitcoin)
+hex | [hex-conservative](https://crates.io/crates/hex-conservative) | [hex-conservative](https://github.com/rust-bitcoin/hex-conservative)
+internals | [bitcoin-internals](https://crates.io/crates/bitcoin-internals) | [rust-bitcoin](https://github.com/rust-bitcoin/rust-bitcoin)
+io | [bitcoin-io](https://crates.io/crates/bitcoin-io) | [rust-bitcoin](https://github.com/rust-bitcoin/rust-bitcoin)
+ordered | [ordered](https://crates.io/crates/ordered) | [rust-ordered](https://github.com/rust-bitcoin/rust-ordered)
+secp256k1 | [secp256k1](https://crates.io/crates/secp256k1) | [rust-secp256k1](https://github.com/rust-bitcoin/rust-secp256k1)
+units | [bitcoin-units](https://crates.io/crates/bitcoin-units) | [rust-bitcoin](https://github.com/rust-bitcoin/rust-bitcoin)
+
+We aim to make usage of these crates secure, ergonomic, and intuitive. We try to enable users of
+these crates to follow software engineering best practices e.g., semver compliance.
+
+If we fall short of these stated aims, or can in anyway improve our support of Bitcoin free software
+projects, please raise issues or contact maintainers directly.
+
+If you write proprietary software and want better integration with these crates by all means raise
+issues on the respective repository. We are interested in all users and your usecase may help us
+improve the libraries, however all development is done in the open and will stay that way.
+
+[0]  (Please note `internals` will explicitly never stabalize.)
+
+## Other Crates
+
+Other crates/repositories hosted here and developed and maintained by the community.
+
+These crates are developed and maintained by a subset of the `rust-bitcoin` devs or other members of
+the community. We are fairly open about hosting any bitcoin-related crates in this org, provided
+that they are actively maintained and:
+
+- Repository contains Bitcoin specific crates (implies Bitcoin Only and not general purpose crates)
+- Repository has at least minimal CI
+- Author has made at least some effort to adhere to practices and standards maintained in the core
```

Comment:
```suggestion
- author has made at least some effort to adhere to practices and standards maintained in the core
```

Reply:
Are you saying all three list items should use lower case starting character? I went the other way and added full stops and used capital. Do say if you think this is wrong - happy to learn.

Reply:
I honestly don't know. But my impression is that the colon, semicolons, and commas they don't end a sentence.
That's the role of the period.
For example, if you are saying something and then you hit a semicolon, you don't start the next word capitalized, right?

The same with the colon.
In a way this

> My things:
>
> - thing one
> - thing two

Is generally written in more formal documents as:

> My things:
>
> - thing one; and
> - thing two.

With semicolons and periods to end.

So this can also be written as:

> My things: thing one; and thing two.

But I don't know if I am correct. Someone or some book taught me this a very long time ago. Can't recollect any details.

Reply:
Cool, I read one web page and updated with my new found knoweldge.

ref: https://editorsmanual.com/articles/lists/